api/envoy/config/core/v3/protocol.proto

```diff
@@ -280,8 +280,24 @@ message Http2ProtocolOptions {
   // the whole HTTP/2 connection is terminated upon receiving invalid HEADERS frame. However,
   // when this option is enabled, only the offending stream is terminated.
   //
+  // This is overridden by HCM :ref:`stream_error_on_invalid_http_messaging
+  // <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.upgrade_configs>`
+  // iff present.
+  //
+  // This is deprecated in favor of stream_error_on_invalid_http_message.
```

Comment:
Can you `:ref:` link to the field below? Given how similar they are name wise, it would be good to have zero ambiguity on which one is being referred to.